examples/app/clustering/main.py

```diff
@@ -1,3 +1,12 @@
+''' A k-nearest neighbors (KNN) chart using datasets from scikit-learn. This
+example demonstrates solving both classification and regression problems.
+
+.. note::
+    This clustering visualization needs the scikit-learn package to run.
+
+.. _K-nearest neighbors algorithm: https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm
```

Comment:
This link alias does not appear to be used anywhere. The docstring could be updated to include it, or the link anchor could be removed